recon_bulk.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Bulk scanning will be implemented in future through python and not a second bash script